apps/condo/pages/meter/create.tsx

```diff
@@ -28,47 +31,78 @@ const CREATE_METER_PAGE_GUTTER: [Gutter, Gutter] = [12, 40]
-    const MeterMessage = intl.formatMessage({ id: 'pages.condo.meter.index.meterTab' })
-    const PropertyMeterMessage = intl.formatMessage({ id: 'pages.condo.meter.index.propertyMeterTab' })
+    const PageTitleAddMeterReadings = intl.formatMessage({ id: 'meter.AddMeterReadings' })
+    const PageTitleAddPropertyMeterReadings = intl.formatMessage({ id: 'meter.AddPropertyMeterReadings' })
+    const PageTitleAddMeters = intl.formatMessage({ id: 'meter.AddMeters' })
+    const PageTitleAddPropertyMeters = intl.formatMessage({ id: 'meter.AddPropertyMeters' })
 
+    const router = useRouter()
+    const { tab } = parseQuery(router.query)
     const { link: { role = {} }, organization } = useOrganization()
+    const canManageMeters = get(role, 'canManageMeters', false)
+
+    const changeRouteToDefaultTab = useCallback(async () => {
+        await updateQuery(router, {
+            newParameters: {
+                tab: METER_TAB_TYPES.meter,
+            },
+        }, { routerAction: 'push', resetOldParameters: true })
+    }, [router])
+
+    useEffect(() => {        
+        if (!Object.values(METER_TAB_TYPES).includes(tab as MeterPageTypes)) changeRouteToDefaultTab()
+    }, [changeRouteToDefaultTab, tab])
+
+    const getPageTitle = useMemo(() => {
+        if (tab === METER_TAB_TYPES.meterReading) return PageTitleAddMeterReadings
+        if (tab === METER_TAB_TYPES.propertyMeterReading) return PageTitleAddPropertyMeterReadings
+        if (tab === METER_TAB_TYPES.meter) return PageTitleAddMeters
+
+        return PageTitleAddPropertyMeters
+    }, [PageTitleAddMeterReadings, PageTitleAddMeters, PageTitleAddPropertyMeterReadings, PageTitleAddPropertyMeters, tab])
 
-    const [tab, setTab] = useState<MeterPageTypes>(METER_PAGE_TYPES.meter)
 
-    const handleTabChange = useCallback((tab: MeterPageTypes) => {
-        setTab(tab)
-    }, [])
 
     return (
         <>
             <Head>
-                <title>{PageTitle}</title>
+                <title>{getPageTitle}</title>
             </Head>
             <PageWrapper>
                 <PageContent>
                     <Row gutter={CREATE_METER_PAGE_GUTTER}>
                         <Col span={24}>
-                            <Typography.Title level={1}>{PageTitle}</Typography.Title>
+                            <Typography.Title level={1}>{getPageTitle}</Typography.Title>
                         </Col>
-                        <Tabs activeKey={tab} onChange={handleTabChange}>
-                            <Tabs.TabPane tab={MeterMessage} key={METER_PAGE_TYPES.meter} />
-                            <Tabs.TabPane tab={PropertyMeterMessage} key={METER_PAGE_TYPES.propertyMeter} />
-                        </Tabs>
                         <Col span={24}>
                             {
-                                tab === METER_PAGE_TYPES.propertyMeter ? (
+                                tab === METER_TAB_TYPES.propertyMeterReading ? (
```

Comment:
mb create a map `tab -> component` and render here instead many ternary operators